core/arch/arm/mm/mobj.c

```diff
@@ -978,10 +978,74 @@ struct mobj *mobj_seccpy_shm_alloc(size_t size)
 	return NULL;
 }
 
+
+#endif /*CFG_PAGED_USER_TA*/
+
+struct mobj_with_fobj {
+	struct fobj *fobj;
+	struct mobj mobj;
+};
+
+static const struct mobj_ops mobj_with_fobj_ops;
+
+struct mobj *mobj_with_fobj_alloc(struct fobj *fobj)
+{
+	if (!fobj)
+		return NULL;
+
+	struct mobj_with_fobj *m = calloc(1, sizeof(*m));
```

Comment:
@jforissier, are you ok with such local var definition? (not at function entry).

Reply:
I'm sorry, this is leftovers from before when it was OK. I'll fix.